base_layer/core/src/helpers/mod.rs

```diff
@@ -45,15 +46,21 @@ pub fn create_orphan_block(block_height: u64, transactions: Vec<Transaction>) ->
         .build()
 }
 
-pub fn create_mem_db() -> BlockchainDatabase<MemoryDatabase<HashDigest>> {
+pub fn create_mem_db(genesis_block: Option<Block>) -> BlockchainDatabase<MemoryDatabase<HashDigest>> /*,Block,UnblindedOutput)*/
```

Comment:
whats with the comment here?

Reply:
Why not `create_mem_db(network: Network)`.

It'll simplify your `match` on line 57 because the compiler will guarantee that a Block is passed in with LocalNet

Reply:
The comment was removed and the function was changed to take in the network rather.